### ISSUE TITLE
Order results displayed in View Experiments

### ISSUE THREAD

**Issue by dcl10** [Biomaterials-for-Medical-Devices-AI/Helix#6]
## Description
Currently, when one selects an experiments for inspection, the order of what is shown in the table does not mach the order of the figures. To change that we need to:

## Tasks
- [x] For Machine Learning type i how results for training set i, followed by result for test set i
- [x]  In the table with the summary results, add a row colour for training sets (light grey) and test sets (white - default)
- [x]  In the plots, the first plot for Machine learning i should be the training set, followed by the test set. So if we have 3 models, for instance Linear Regression, Random Forest and XGboost, the results in the table should be Linear Regression training set, Linear Regression test set, Random Forest training set, Random Forest test set, etc. The same order for the plots.

**Comment by Grazziela** [comment]
The tasks are complete although at the moment the test set is still displayed before the training set, due to the order by which plots is being created. @dcl10  will look into that in the next sprint and the code has been modified so we can easily change the order of train/test display.
